Library/Homebrew/cmd/shellenv.sh

```diff
@@ -5,31 +5,35 @@
 #:  The variables `HOMEBREW_PREFIX`, `HOMEBREW_CELLAR` and `HOMEBREW_REPOSITORY` are also exported to avoid querying them multiple times.
 #:  Consider adding evaluation of this command's output to your dotfiles (e.g. `~/.profile`, `~/.bash_profile`, or `~/.zprofile`) with: `eval $(brew shellenv)`
 
+# need not double quoting ${PPID}
+# HOMEBREW_CELLAR and HOMEBREW_PREFIX are set by extend/ENV/super.rb
+# HOMEBREW_REPOSITORY is from the user environment
+# shellcheck disable=SC2248,SC2154
 homebrew-shellenv() {
-  case "$(/bin/ps -p $PPID -c -o comm=)" in
+  case "$(/bin/ps -p ${PPID} -c -o comm=)" in
```

Comment:
```suggestion
  case "$(/bin/ps -p "${PPID}" -c -o comm=)" in
```

which should remove need for `SC2248` disable